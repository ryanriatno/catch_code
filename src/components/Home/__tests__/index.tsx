import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import Index from '../index'

describe("<ProductCard /> test case", () => {
  test("Render Header without error", () => {
    const { container } = render(<Index  />);
    expect(container).toMatchSnapshot();
  });
});