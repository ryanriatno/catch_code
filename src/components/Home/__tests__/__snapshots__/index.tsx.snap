// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductCard /> test case Render Header without error 1`] = `<div />`;
